source/includes/fact-verifier-memory.rst

```diff
@@ -0,0 +1,16 @@
+
+Verification requires 10 GB of memory plus an additional 500 MB
+for every 1 million documents in the migration. 
+
+If the server ``mongosync`` runs on doesn't have sufficient
+memory available, the ``/start`` endpoint returns an error. 
+
+To use the verifier when this occurs, you must increase the
```

Comment:
This block is leftover from the OOM error. Can we remove?

Reply:
We can keep the note around the initial failure check, which I think is what this is

Reply:
Reworked